### PR TITLE
New staff!

### PR DESCRIPTION
**Pull request by davidharrison92**

### CODE REVIEW

File: itea.html [hunk 1]
```diff
@@ -52,7 +52,7 @@ <h1>IS Tea Round Cheat Sheet</h1>
 			<tr><td><strong>Dave</strong></td><td> HPA, else some lager like San Miguel. Estrella in the Robin Hood. </td></tr>
 			<tr><td><strong>Luke</strong></td><td> Bitter. If there's one with an amusing name, go for that.</td></tr>
 			<tr><td><strong>Gary</strong></td><td> Usually Guinness. After playing badders, a refreshing pint of Lager.</td></tr>
-			<tr><td><strong>James</strong></td><td> <del>Pepsi</del> Coca Cola</td></tr>
+			<tr><td><strong>Chris</strong></td><td>Usually Guinness. Also enjoys</td></tr>
```
Comment (gchristian17) [inline itea.html]: You've got Roy and Chris in the wrong order in this section

File: itea.html [hunk 2]
```diff
@@ -35,8 +35,8 @@ <h1>IS Tea Round Cheat Sheet</h1>
 			<tr><td><strong>Dave</strong></td><td> Almost always has WHITE TEA WITH ONE SUGAR (but if he looks hung-over, ask if he'd like a morning WHITE COFFEE WITH ONE SUGAR) </td></tr>
```
Comment (gchristian17) [inline itea.html:35]: You could have line breaks after tr and td so that no line gets too long. (Standard would be max 80 characters/line)

Comment (davidharrison92) [reply itea.html:35]: ¯\_(ツ)_/¯

File: itea.html (hunk 1 above)
Comment (davidharrison92) [reply itea.html]: new commit.

**Pull request merged**